.github/workflows/server-test-template.yml

```diff
@@ -77,7 +77,7 @@ jobs:
             server/report.xml
       - name: Publish test report
         id: report
-        uses: mikepenz/action-junit-report@a427a90771729d8f85b6ab0cdaa1a5929cab985d # v5.0.0
+        uses: mikepenz/action-junit-report@992d97d6eb2e5f3de985fbf9df6a04386874114d # v5.1.0
```

Comment:
Good to see that the PR I've created with `retried` output is now included in `v5.1.0`. Will prepare a follow PR to update its usage.